io.github.orontee.Argos.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "io.github.orontee.Argos",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "41",
```

Comment:
```suggestion
    "runtime-version" : "42",
```

Reply:
Looks good to me. Thanks! Applied upstream, https://github.com/orontee/argos/commit/ab31a584b6ab8a68f7772d84536b78c89f6abced